packages/drivers/odsp-driver/src/test/odspError.spec.ts

```diff
@@ -58,8 +58,7 @@ describe("Odsp Error", () => {
                 "message should contain original message");
             assert.notEqual(-1, networkError.message.indexOf("testStatusText"),
                 "message should contain Response.statusText");
-            assert.notEqual(-1, networkError.message.indexOf("default"),
-                "message should contain Response.type");
+            assert((networkError as any).type === "default", "message should contain Response.type");
```

Comment:
nit: assertion text should indicate specifically that `type === "default"`, not just that it exists.